lib/Agrammon/OutputFormatter/JSON.pm6

```diff
@@ -63,19 +66,17 @@ sub get-data($model, $outputs, $include-filters, $language?, $prints?) {
     return @records.sort(+*.<order>);
 }
 
-sub make-record($fq-name, $output, $model, $raw-value, $var, $order, $instance-id?, :$language, :$prints, :@filters) {
+sub make-record($fq-name, $output, $model, $raw-value, $var, $order, $instance-id?, :$language, :@filters, :@print-set) {
+    my $var-print = $model.output-print($fq-name, $output);
+    next unless $model.should-print($fq-name, $output, @print-set);
```

Comment:
Probably more efficient to swap these lines around?

Reply:
Grr, I thought I had caught all of them ... fixed.